StripePaymentSheet/StripePaymentSheet/Source/Internal/Link/Extensions/STPAnalyticsClient+Link.swift

```diff
@@ -36,28 +36,6 @@ extension STPAnalyticsClient {
         self.logPaymentSheetEvent(event: .linkSignupFailure)
     }
 
-    // MARK: - 2FA
-
-    func logLink2FAStart() {
-        self.logPaymentSheetEvent(event: .link2FAStart)
-    }
-
-    func logLink2FAStartFailure() {
-        self.logPaymentSheetEvent(event: .link2FAStartFailure)
-    }
-
-    func logLink2FAComplete() {
-        self.logPaymentSheetEvent(event: .link2FAComplete)
-    }
-
-    func logLink2FAFailure() {
-        self.logPaymentSheetEvent(event: .link2FAFailure)
-    }
-
-    func logLink2FACancel() {
-        self.logPaymentSheetEvent(event: .link2FACancel)
-    }
```

Comment:
Removing these means that.. i guess we can go into STPAnalyticEvent.swift and remove the following:
```
    // MARK: - Link 2FA
    case link2FAStart = "link.2fa.start"
    case link2FAStartFailure = "link.2fa.start_failure"
    case link2FAComplete = "link.2fa.complete"
    case link2FACancel = "link.2fa.cancel"
    case link2FAFailure = "link.2fa.failure"
```